### DISCUSSION TITLE
skip the schema name when using atlas migrate diff

### DISCUSSION THREAD

**Discussion by delamainer** [ariga/atlas#2482] [Q&A]
Is there a way to not include the schema name in the generated SQL file?

When I run `atlas migrate diff`, I see that the name of the schema (in this case, it's `public`) is included in the generated file.
Is there a way to exclude a schema from SQL files and provide it when I run `atlas migrate apply`.

```sql
CREATE TABLE "public"."user"
(
    "id"         uuid                   NOT NULL DEFAULT gen_random_uuid(),
    "email"      character varying(255) NOT NULL,
    "password"   character varying(255) NOT NULL
    PRIMARY KEY ("id")
);
```

**Comment by a8m** [comment]
Yes, change the dev-url to "schema scope". i.e., add `search_path=public` to the URL. More info exists on our website: https://atlasgo.io/versioned/diff#generate-migrations-from-hcl-schemas
